artifacts/definitions/Windows/Events/Windows.Events.Trackaccount.yaml

```diff
@@ -0,0 +1,43 @@
+name: Windows.Events.Trackaccount
+description: |
+  Artifact to detect account usage by monitoring event id 4624. This is useful for tracking attacker activity. If you want to receive Slack notifications you can enable the server_event artifact named 'Server.Alerts.Trackaccount'
+
+author: Jos Clephas
+
+type: CLIENT_EVENT
+
+parameters:
+  - name: eventLog
+    default: C:\Windows\system32\winevt\logs\Security.evtx
+  - name: UserRegex
+    default: 'admin|user'
+  - name: LogonTypeRegex
+    default: '[3,10]'
+
+sources:
+  - precondition:
+      SELECT OS From info() where OS = 'windows'
+    queries:
+      - LET files = SELECT * FROM glob(globs=eventLog)
+
+      - LET LogonTypeVar <= parse_json_array(data=LogonTypeRegex)
```

Comment:
You dont need to do this explicitly - just specify the type of the parameter as json_array and it will be done automatically.

The advantage is that we might add a GUI element for a json array in future (there is none atm).